cinema/serializers.py

```diff
@@ -1,24 +1,94 @@
 from rest_framework import serializers
+from rest_framework.validators import UniqueValidator
 
-from cinema.models import Movie
+from cinema.models import Actor, Genre, CinemaHall, Movie
+
+
+class ActorSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    first_name = serializers.CharField(max_length=50)
+    last_name = serializers.CharField(max_length=50)
```

Comment:
I'm not sure, but I think it is time to use serializers.ModelSerializer instead of Serializer
so your "serializers.py" code will become much shorter
4 strings almost for each class, except Movie with it ManyToMany realations